scripts/profiler/rtt_nordic_profiler_host.py

```diff
@@ -100,26 +97,28 @@ def shutdown(self):
     def disconnect(self):
         self.stop_logging_events()
         # read remaining data to buffer
-        try:
-            buf = self.jlink.rtt_read(self.config['rtt_data_channel'],
-                                      self.config['rtt_read_chunk_size'],
-                                      encoding=None)
+        while True:
```

Comment:
This hunk looks like changed semantics, not just a linter change. Am I missing something?

Reply:
I just wanted to make sure that `rtt_read` is in the try: catch: to properly detect errors. The change is small, so maybe we can leave it as part of this commit ?

Reply:
OK, sure